src/order-book/api.ts

```diff
@@ -126,15 +125,15 @@ export type GetOrdersRequest = {
  * @see {@link OrderBook API https://github.com/cowprotocol/services}
  */
 export class OrderBookApi {
-  public context: ApiContext & RequestOptions
```

Comment:
Why remove this?

Reply:
`RequestOptions` was merged into `ApiContext`

Reply:
<img width="334" alt="image" src="https://github.com/cowprotocol/cow-sdk/assets/7122625/72722543-7ca9-4d1b-b9b6-4289f174b02f">